cms/models.py

```diff
@@ -146,7 +146,9 @@ def get_context(self, request):
         context = super(ProgramPage, self).get_context(request)
 
         courses_info = {}
-        for course in self.program.course_set.all():
+        for course in self.program.course_set.all().order_by(
```

Comment:
Python won't store the order of items in the dictionary (at least, it's not guaranteed that this will be true in the future). Can you change `courses_info` to be an `OrderedDict` above?